client/components/theme/index.jsx

```diff
@@ -172,8 +176,12 @@ const Theme = React.createClass( {
 								context: 'singular noun, the currently active theme'
 							} ) }</span>
 						}
-						{ price && ! purchased &&
-							<span className="theme-badge__price">{ price }</span>
+						{
+							config.isEnabled( 'jetpack/pijp' )
+							? ! this.props.isJetpack && price && ! purchased &&
```

Comment:
Okay, so this sort of mixes up our previous abstraction -- previously, we'd only hide the price tag if a premium theme had been purchased, be it individually, or as part of a plan.

But maybe we should consider the UX aspects of this a bit more? Not displaying a price and only changing the "Purchase" menu item to "Upgrade to activate" and have it point to the plans page doesn't seem like great UX. (E.g. it's not visible at a glance which themes are readily available, and which need an upgrade.) Maybe we should continue to visually indicate that? /cc @folletto for design advice